manager/src/lib.rs

```diff
@@ -154,40 +134,71 @@ impl CronManager {
     }
 
     /// Gets a set of tasks.
-    /// Default: Returns the next executable set of tasks hashs.
+    /// Default: Returns the next executable set of tasks hashes.
     /// 
     /// Optional Parameters:
     /// "offset" - An unsigned integer specifying how far in the future to check for tasks that are slotted.
     /// 
     /// ```bash
-    /// near view cron.testnet get_tasks --accountId YOU.testnet
+    /// near view cron.testnet get_tasks
     /// ```
-    pub fn get_tasks(&self, offset: Option<u64>) -> Vec<Vec<u8>> {
+    pub fn get_tasks(&self, offset: Option<u64>) -> Vec<Base64VecU8> {
         let current_slot = self.get_slot_id(offset);
-        let default_vec: Vec<Vec<u8>> = Vec::new();
 
-        // get tasks based on current slot
-        self.slots.get(&current_slot).unwrap_or(default_vec)
+        // Get tasks based on current slot.
+        // (Or closest past slot if there are leftovers.)
+        let slot_ballpark = self.slots.floor_key(&current_slot);
```

Comment:
Hey @TrevorJTClarke this is the part that I think you might take issue with. Don't want this to get buried in the changes. 
I imagine there will be "gaps" in slots. Like perhaps lots of people want to execute around midnight on PST on the first of the month, but way less slots on the 3rd of the month. This basically tries to get the desired slot, but if there isn't an exact key for it, it'll see what the **closest, past, valid** slot is. So basically if there were leftover tasks in a slot, it would find that.

Reply:
no, this solves a ton of the edge case things that would come up with my approach -- love it! :heart: